lib/client-assets.php

```diff
@@ -410,7 +410,7 @@ function gutenberg_register_packages_styles( $styles ) {
 		$styles,
 		'wp-edit-site',
 		gutenberg_url( 'build/edit-site/style.css' ),
-		array( 'wp-components', 'wp-block-editor', 'wp-editor', 'wp-edit-blocks', 'wp-commands', 'wp-preferences' ),
+		array( 'wp-components', 'wp-block-editor', 'wp-editor', 'common', 'forms', 'wp-commands', 'wp-preferences' ),
```

Comment:
Do you know why edit-blocks needs common and forms?

Reply:
Inside the canvas it's currently needed for placeholders (another reason to do the shadow DOM finally).
Outside the canvas we depend on it for all UI. Which is a bit surprising because I'd expect the components package to handle everything.

Reply:
> Inside the canvas it's currently needed for placeholders (another reason to do the shadow DOM finally).

Same for me inside the canvas. I expect components to handle everything, so I wonder why we need it there at all. Maybe we don't need it either and it just there for historic reasons?